hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/record/entity/EntityRecordItemListener.java

```
@@ -310,7 +316,7 @@ private EntityId getAccountId(AccountID accountID) {
                 var alias = DomainUtils.toBytes(accountID.getAlias());
                 return entityRepository.findByAlias(alias)
                         .map(id -> EntityId.of(id, EntityType.ACCOUNT))
-                        .orElseThrow(() -> new InvalidDatasetException("AccountID not present for alias: " + Hex.encodeHexString(alias)));
+                        .orElseThrow(() -> new AliasNotFoundException(Hex.encodeHexString(alias)));
```

For Hedera managed mirror nodes we probably want to fail fast on bad data so we can catch errors early. We should have some property that controls whether to fail fast on partial data issues. We should make it generic so it works for EVM address and any other fields that have problems with partial mirror nodes. 

It should be an enum so it can support different actions. Something like `hedera.mirror.importer.parser.record.partialDataAction=DEFAULT|ERROR|SKIP`. Where `DEFAULT` will replace the unknown values with defaults appropriate for its data type like `EntityId.EMPTY`, 0, etc. `ERROR` is the current action of throwing an exception and halting processing. `SKIP` would omit the entire transaction.

---

We could even dynamically detect if it's a partial mirror node by checking if first record item entry is the zero hash or not, then set this to DEFAULT or SKIP automatically.